org.eclipse.jdt.core/model/org/eclipse/jdt/core/ICompilationUnit.java

```diff
@@ -845,4 +847,13 @@ default Map<String, String> getOptions(boolean inheritJavaCoreOptions) {
  */
 @Override
 void restore() throws JavaModelException;
+
+/**
+ * Can be used to synchronize the {@link CompilationUnitElementInfo} timestamp to its corressponding {@link IResource}s timestamp.
```

Comment:
`CompilationUnitElementInfo` is internal non API type, it shouldn't appear in API javadoc